src/components/App.jsx

```diff
@@ -37,6 +37,10 @@ function App() {
          <Home />
       </Route>
 
+      <Route path="/home">
```

Comment:
not sure if this path is necessary since home is the default page.

Reply:
The link to floor plan for each data category will have "/home/:cateId" if "/home" is not assigned to some page it will confuse user. "/home" will link to floor plan with default category of user's choice.